### DISCUSSION TITLE
How to upgrade superset but keep superset_config.py

### DISCUSSION THREAD

**Discussion by timnick-snow** [apache/superset#18860] [Q&A / Help] [answered]
![image](https://user-images.githubusercontent.com/33312542/155249017-23d302b0-44a0-4d38-bc13-ac3da18a0b00.png)

I edited superset_config.py and added it to /app/pythonpath in the container
after run `docker-compose down`,  containers will be removed.
then run `docker-compose up`.  All files in the container under /app/pythonpath are lost

so .  How to upgrade superset but keep superset_config.py

**Comment by jacobcroope** [comment, marked as answer]
Not sure if this is what you are looking for but I had to put my specific configuration files in the docker config file to make them persistent. See [Readme](https://github.com/apache/superset/tree/master/docker)

**Reply by timnick-snow** [reply]
That's very helpful. Thank you !
